docs/releases/4.0.txt

```diff
@@ -58,6 +58,12 @@ For example::
 Functional unique constraints are added to models using the
 :attr:`Meta.constraints <django.db.models.Options.constraints>` option.
 
+``scrypt`` password hasher
+--------------------------
+
+The new :ref:`scrypt password hasher <scrypt-usage>` is substantially more
```

Comment:
If we are recommending it like this (do we really want to?) we should also switch the default (at least in the project template) to be scrpt.

Reply:
> If we are recommending it like this (do we really want to?) 

I based it on @alex's opinion in the ticket description. Maybe too enthusiastic :thinking:  

> we should also switch the default (at least in the project template) to be scrpt.

I was thinking about it, but it requires OpenSSL 1.1+ so it may be too early to the change the default for new projects.

Reply:
Then I'd add least add a note why it is not the default yet (maybe also tone down the wording a bit). Also the memory requirements can cause problems for some…

Reply:
I adjusted release notes, any suggestions are welcome.